metadata_service/proxy/atlas_proxy.py

```diff
@@ -72,37 +74,61 @@ def _extract_info_from_uri(self, *, table_uri: str) -> Dict:
         result = pattern.match(table_uri)
         return result.groupdict() if result else dict()
 
-    def get_user_detail(self, *, user_id: str) -> Union[UserEntity, None]:
-        pass
-
-    def get_table(self, *, table_uri: str) -> Table:
+    def _get_table_entity(self, *, table_uri: str) -> Tuple:
         """
-        Gathers all the information needed for the Table Detail Page.
-        It tries to get the table information from
+        Fetch information from table_uri and then find the appropriate entity
+        The reason, we're not returning the entity_unique_attribute().entity
+        directly is because the entity_unique_attribute() return entity Object
+        that can be used for update purposes,
+        while entity_unique_attribute().entity only returns the dictionary
         :param table_uri:
         :return:
```

Comment:
Could you add some explanation about the return type's schema to know what data is available? (It's because the reader will only know by looking at the code who consumes the data, but still hard to have full picture)